tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -277,54 +277,74 @@ def _setup_node(self) -> None:
         log.info("Resource Group: %s", self.context.vm.resource_group)
         log.info("")
 
-        self.context.ssh_client.run_command("mkdir -p ~/bin/tests_e2e/tests; touch ~/bin/agent-env")
```

Comment:
Now we prepare a tarball on the orchestrator machine that is already structured in the way it needs to be setup on the test VMs, instead of using SSH to create/copy these files. This should make easier to include additional files that need to be copied to the test VMs.